### PR TITLE
refactor: admob

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
SDK:
- Update the response type of ads to string, Reward.
- Device OS detection. The value could be `Android`, `iOS` or `Unknown`.

Bridge:
- Update the response type of ads to string, Reward.
- Platform detection. The value could be `Android` or `iOS`.

# Checklist
- [x] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-bridge/src/android/android-bridge.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ class AndroidExcecutor implements PlatformExecutor {
       JSON.stringify({ action, param, id: callback.id })
     );
   }
+
+  getPlatform(): string {
+    return 'Android';
```
Comment (corycaywood) [inline js-miniapp-bridge/src/android/android-bridge.ts:25]: Can we make an `enum` for this? So we could use something like `Platform.ANDROID` here.

**Review by corycaywood — APPROVED** [review]
Looks good 👍  Just left a suggestion

**Review by Abhijit-majumder — APPROVED** [review]
Looks Good 👍 
Like Cory mentioned, please put possible platforms to enum
Thanks

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**